flink/flink-base/src/main/java/io/tidb/bigdata/flink/tidb/TypeUtils.java

```diff
@@ -106,7 +106,7 @@ public static DataType getFlinkType(org.tikv.common.types.DataType dataType) {
   /**
    * transform TiKV java object to Flink java object by given Flink Datatype
    *
-   * @param object TiKV java object
+   * @param object   TiKV java object
```

Comment:
There is an extra space here @marsishandsome

Reply:
after import `intellij-java-google-style.xml` to Intellij, `align parameter descriptions` is set to true.

![image](https://user-images.githubusercontent.com/5574887/107019761-04268a00-67dd-11eb-8ed7-7978e0aacf51.png)

Reply:
Hm, good to know it.